dashboard/app/models/level_concept_difficulty.rb

```diff
@@ -0,0 +1,35 @@
+# == Schema Information
+#
+# Table name: level_concept_difficulties
+#
+#  id                    :integer          not null, primary key
+#  level_id              :integer
+#  created_at            :datetime         not null
+#  updated_at            :datetime         not null
+#  sequencing            :integer
+#  debugging             :integer
+#  repeat_loops          :integer
+#  repeat_until_while    :integer
+#  for_loops             :integer
+#  events                :integer
+#  variables             :integer
+#  functions             :integer
+#  functions_with_params :integer
+#  conditionals          :integer
```

Comment:
How often do we expect these columns to change (e.g., adding a new concept, or changing its name, etc)? Hopefully it's not often, otherwise this will add an extra maintenance burden (need to create and deploy a new db migration every time this list is modified)